CSS/Styles.css

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
- [ ] [OPTIONAL] If you remove margin from body, it will be good design for mobile version.

![image](https://user-images.githubusercontent.com/63915024/182454114-67228188-9885-493a-88ea-3ffb0ab43b02.png)